R/utils.R

```diff
@@ -170,7 +171,7 @@ post_x <- function(x, params) {
   }
   if (sum(idx_integer) > 0L) {
     x[, idx_integer] <- lapply(idx_integer, function(j) {
-      if (is.numeric(x[[j]])) {
+      if (is.numeric(x[[j]]) & round) {
```

Comment:
For round==FALSE, the numeric value is converted to character first, effectively (I think) rounding down.